builder/builder.go

```diff
@@ -26,11 +31,6 @@ import (
 	"path/filepath"
 	"runtime"
 	"sync"
-
-	"github.com/clearlinux/mixer-tools/config"
-	"github.com/clearlinux/mixer-tools/helpers"
-	"github.com/clearlinux/mixer-tools/swupd"
-	"github.com/pkg/errors"
```

Comment:
Not sure about this reordering of imports... normally, the mixer-tools and third-party package imports are imported like this to visually separate the imports from the standard library. See other mixer-tools source files as a reference.

Reply:
From what I recollect 'gofmt -w' did this.

Reply:
Hmm, interesting, I tried running `gofmt -w builder/builder.go` on the current master branch, but see no modifications to the file...

Reply:
This happens when you "gofmt -w" after importing "encoding/json" in the empty line [29](https://github.com/clearlinux/mixer-tools/blob/master/builder/builder.go#L29) between "sync" and "github.com/clearlinux/mixer-tools/config"

Anyways, I have changed it as per your comment.

Reply:
Formatting works in chunks w/no empty lines in between, so if you clump "fmt" and "github.com/.." it will sort them in the same paragraph, but if it's separated with an empty line it will treat paragraphs/chunks of includes as separate and sort them individually.